Doc/library/re.rst

```diff
@@ -786,13 +786,17 @@ form.
 
 .. function:: escape(string)
 
-   Escape all the characters in pattern except ASCII letters, numbers and ``'_'``.
+   Escape special characters in a string.
```

Comment:
In #1048 the `string` parameter name was replaced by `pattern` and it is referred as `*pattern*` in the description.